diff --git a/rog-aura/data/aura_support.ron b/rog-aura/data/aura_support.ron
index 8ab3b34e..d3f83d5b 100644
--- a/rog-aura/data/aura_support.ron
+++ b/rog-aura/data/aura_support.ron
@@ -998,4 +998,13 @@
         advanced_type: r#None,
         power_zones: [Ally],
     ),
-])
\ No newline at end of file
+    (
+        device_name: "GA403W",
+        product_id: "",
+        layout_name: "ga401q",
+        basic_modes: [Static, Breathe, RainbowCycle, RainbowWave, Pulse],
+        basic_zones: [],
+        advanced_type: r#None,
+        power_zones: [Keyboard],
+    ),
+])
